.sass-lint.yml

```diff
@@ -5,6 +5,7 @@ options:
   merge-default-rules: true
 
 rules:
+  class-name-format: 0
```

Comment:
It seemed doubtful that we were going to rename a bunch of our CSS selectors this late in the game, so we can probably just ignore this rule for the time being.